resample_states_verification/NOTES.md

```diff
@@ -0,0 +1,58 @@
+
+Testing of changes to use mock GET_PITCH commands
+=================================================
+
+Used the timelines code to make a new test database and cmd states for 2010.  This made
+the archive by stepping through the days, which we don't care about for this, but, in the
+timelines repo:
+
+
+    import timelines_test
+    timelines_test.all_2010(cmd_state_ska='/data/fido/ska')
+
+I then used fetch_states against the flight cmd_states.h5 and the testing cmd_states.h5 to
+create dat files limited to the columns that really should not be changing due to this
+update.
+
+    from Chandra.cmd_states.get_cmd_states import fetch_states
+    from astropy.table import Table
+    
+    official = fetch_states(start='2010:005', stop='2010:360', vals=['obsid', 'power_cmd',
+      'si_mode', 'pcad_mode', 'vid_board', 'clocking', 'fep_count', 'ccd_count', 'simpos',
+      'simfa_pos', 'hetg', 'letg'], server='/proj/sot/ska/data/cmd_states/cmd_states.h5')
+    Table(official).write('official_minus_point.dat', format='ascii')
+    
+    test = fetch_states(start='2010:005', stop='2010:360', vals=['obsid', 'power_cmd',
+      'si_mode', 'pcad_mode', 'vid_board', 'clocking', 'fep_count', 'ccd_count', 'simpos',
+      'simfa_pos', 'hetg', 'letg'], server='t/all_2010/cmd_states.h5')
+    Table(test).write('test_minus_point.dat', format='ascii')
+    
+    meld official_minus_point.dat test_minus_point.dat
+
+The small set of small timing diffs are acceptable on the non-pointing-related states.
```

Comment:
Not immediately.  This is one of the reasons that I just I included the
actual data/test output files so that you could see the diffs yourself with
a

"meld official_minus_point.dat test_minus_point.dat"

I'm diffing here against what is in the official sybase database now for
the year 2010.  I suppose a better test would have been to make a similar
dat file using the cmd_states code currently in master as the reference.  I
don't think I have a way to know which version of the code and which
version of the Ska runtime made the states that are in the database for
this interval.